kedro/framework/cli/starters.py

```diff
@@ -91,6 +91,9 @@ def new(
             "Cannot use the --directory flag without a --starter value."
         )
 
+    starter_name = (
```

Comment:
Maybe include a little inline comment why we're doing this?

Reply:
Done!